django_project/ford3/static/css/wizard_form.css

```diff
@@ -50,4 +50,45 @@
 
 .form-check-inline {
     padding-top: 10px;
-}
\ No newline at end of file
+}
```

Comment:
Could we reuse CSS variables defined in https://github.com/kartoza/ford3/blob/32eb05d624608ba67afd244941956b109640562e/django_project/core/base_static/css/stylesheet.css ?

Reply:
For the common element like buttons, I use that css. But this is for specific element for this form.